### DISCUSSION TITLE
一个报错

### DISCUSSION THREAD

**Discussion by xiaoBingge114514** [xbzstudio/Timeless-Sydney#64] [Q&A] [answered]
D:\Python\python.exe E:\Desktop\EdgeGPT\BingAI-Client-BingAI-Client1.7.1-zhengshiban-\EasyStart.py 
假如您想高度自定义配置(如端口)，请勿使用简单启动！
您的服务器将在 80 端口运行
是否对公网开放?(需要有公网ip)(y/n):n

E:\Desktop\EdgeGPT\BingAI-Client-BingAI-Client1.7.1-zhengshiban-\run>python ../BingServer.py 
{'AllowConnect': False, 'Port': 80}
Traceback (most recent call last):
  File "E:\Desktop\EdgeGPT\BingAI-Client-BingAI-Client1.7.1-zhengshiban-\BingServer.py", line 62, in <module>
    def getStyleEnum(style: str) -> EdgeGPT.ConversationStyle:
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'EdgeGPT' has no attribute 'ConversationStyle'

进程已结束,退出代码0
![image](https://github.com/xbzstudio/BingAI-Client/assets/105868119/9d143a8d-8921-4bd9-9369-7f37513486e9)
EdgeGPT 0.10.12

**Comment by xbzstudio** [comment, marked as answer]
降低EdgeGPT版本。EdgeGPT的新版本出了一个反人类更新，现在ConversationStyle这些东西都包含在EdgeGPT.EdgeGPT里。
